### DISCUSSION TITLE
Stats from past day

### DISCUSSION THREAD

**Discussion by JamesPHawk** [cwendt94/espn-api#315] [ESPN API] [answered]
First of all, I love this library; it's been quite easy to use and very navigable.

With that said, is there some way to get a basketball player's stats from the past day? The stats dict returns last_7/15/30, 2021 stats, 2022 stats, 2022 projections, and strangely the 2021 projections. Not sure if that's always been the case but I can't find a workaround. Any ideas?

**Comment by JamesPHawk** [comment]
Ok so I dug into it a bit more and I think ESPN has a window where the API will return last night's game before clearing it out to prepare for the upcoming day's games. Just a hypothesis, could be wrong.

Also learned that ESPN tags games with a unique ID. For example, today's Lakers @ Nets game has and ID of 401360537.

https://www.espn.com/apis/devcenter/docs/scores.html for more on that.

**Reply by cwendt94** [reply]
Let me look into it! I will have to check the data ESPN returns in some of the API calls. We should be able to get the stats for every game played for a player.

**Comment by cwendt94** [comment, marked as answer]
Thanks for brining this back up, on espn fantasy website they only allow you to filter last 7/15/30 days but I will check the data to see if they provide any other stats.

When you click on the player it then shows the players stats for the specific games, so might have to get that information from functionality similar to [footballs player_info](https://github.com/cwendt94/espn-api/wiki/League-Class) call.

**Reply by cwendt94** [reply]
`player_info` functionality is now in v0.26.0 and will fill out stats for every game for the player. See [function overview](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) and [stats example](https://github.com/cwendt94/espn-api/wiki/Player-Class-Basketball) in the player class
